api/cpp/tests/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ macro(slint_test NAME)
     target_compile_definitions(test_${NAME} PRIVATE
         SOURCE_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/\"
     )
+    # Use debug version of run-time library to enable MSVC iterator debugging
+    set_property(TARGET test_${NAME} PROPERTY MSVC_RUNTIME_LIBRARY MultiThreadedDebugDLL)
```

Comment:
Should it perhaps be `MultiThreadedDebug`  (without the DLL)  ?

Reply:
I don't think so, I'm pretty sure we want to use dynamic linkage against the Visual Studio C Run-time, instead of static. slint_cpp.dll is also dynamically linked.